### DISCUSSION TITLE
Adding New Country Dataset (Azerbaijan)

### DISCUSSION THREAD

**Discussion by rafig-dev** [ria-com/nomeroff-net#239] [Q&A]
Hello Nomeroff Team,

I was curious if it's possible to add Azerbaijan License Plates to your Dataset List. Can you please provide detailed information about the requirements to help you expand your dataset with Azerbaijan License Plates.

Thanks in advance,
Rafig

**Comment by ApelSYN** [comment]
We can add support for your country, for this we need a dataset of 10,000+ photos of cars with Azerbaijan license plates.

After preprocessing the dataset, the dataset will need to be checked using our admin panel, after that we will train az-ocr.

The photo can be obtained by parsing thematic sites or purchasing a dataset on specialized sites, for example https://platesmania.com/
